src/pages/CodPage.tsx

```diff
@@ -0,0 +1,20 @@
+import "../components/modal/modal.scss";
+import Button from "../components/Button";
+import PasswordAndConfirm from "../components/PasswordAndConfirm";
+
+const CodPage=()=>{
+
+  return(
+    <div className='field'>
+      <div className='modal'>
+        <h2 className='title'>Now a code will come to your phone. Enter it in a line.</h2>
+        <PasswordAndConfirm minSymbol={5} maxSymbol={5} isConfirm={false} error={'Code must contain 5 characters'}/>
```

Comment:
Please store minSymbol={5} maxSymbol={5} in separate constant, e.g.

const PASSWORD_LENGTH = 5

See examples of constants in current project components. As rule they are stored in separate file.

Why separate constants? Read here: https://ru.hexlet.io/blog/posts/magic-numbers